src/test/scala/com/microsoft/partnercatalyst/fortis/spark/SparkSpec.scala

```diff
@@ -1,18 +1,27 @@
-package com.microsoft.partnercatalyst.fortis.spark.streamprovider
+package com.microsoft.partnercatalyst.fortis.spark
 
-import org.apache.spark.{SparkConf, SparkContext}
+import java.time.Instant
+import java.util.UUID
+import java.util.UUID.randomUUID
+
+import com.datastax.spark.connector._
+import com.microsoft.partnercatalyst.fortis.spark.dto.{Analysis, AnalyzedItem}
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.{CassandraConfig, CassandraSink}
+import com.microsoft.partnercatalyst.fortis.spark.streamprovider.{ConnectorConfig, StreamFactory, StreamProvider, UnsupportedConnectorConfigException}
 import org.apache.spark.rdd.RDD
-import org.apache.spark.streaming.{Seconds, StreamingContext}
 import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import org.apache.spark.{SparkConf, SparkContext}
 import org.scalatest.{BeforeAndAfter, FlatSpec}
 
 import scala.collection.mutable
 
-class StreamProviderSpec extends FlatSpec with BeforeAndAfter {
+class SparkSpec extends FlatSpec with BeforeAndAfter {
```

Comment:
As per the comment on 629e5d3, we can only have a single Spark Context running per JVM so I merged all the tests for now (they were previously split but it's non-trivial to get that to work). If this becomes an issue, we can spend the time to figure out how to split the tests.